audiorecorder/src/main/java/org/odk/collect/audiorecorder/recorder/MediaRecorderWrapper.kt

```diff
@@ -0,0 +1,63 @@
+package org.odk.collect.audiorecorder.recorder
+
+import android.media.MediaRecorder
+
+/**
+ * Allows faking/stubbing/mocking with our interactions with Android's MediaRecorder
+ */
+
+interface MediaRecorderWrapper {
```

Comment:
This interface brought to you by a slightly funky looking `ShadowMediaRecorder` and the "don't mock things you don't own" rule.